source/macos/AdaptiveCards/AdaptiveCards/Views/ACRChoiceButton.swift

```diff
@@ -6,18 +6,25 @@ protocol ACRChoiceButtonDelegate: NSObjectProtocol {
 
 class ACRChoiceButton: NSView, NSTextFieldDelegate, InputHandlingViewProtocol {
     weak var delegate: ACRChoiceButtonDelegate?
+    public var choiceSetButtonConfig: ChoiceSetButtonConfig = .default
     public var buttonValue: String?
```

Comment:
```suggestion
    public var buttonValue: String?
```
You don't need this. 
Add this to the `RenderConfig` and consume it from there.